src/http.c

```diff
@@ -77,6 +77,24 @@ int http_connect( http_t *conn, int proto, char *proxy, char *host, int port, ch
 		conn->local_if, conn->headers ) == -1 )
 		return( 0 );
 
+	//For SSl through proxy
+	if( conn->proxy && PROTO_IS_SECURE(conn->proto))
+	  {
+	    char proxybuffer[2000];
+	    sprintf(proxybuffer,"CONNECT %s HTTP/1.1\r\nHost: %s\r\nUser-Agent: Axel\r\n\r\n",conn->host,conn->host);
```

Comment:
Space, before opening and after closing brackets. Also, there should be a whitespace after every comma (between arguments). Please fix this everywhere in the patch.